lib/definitions/project.d.ts

```diff
@@ -140,6 +140,7 @@ interface IPlatformProjectServiceBase {
 
 interface IBuildForDevice {
 	buildForDevice: boolean;
+	release: boolean;
```

Comment:
The idea of `IBuildForDevice` interface is to be used whenever `buildForDevice` option is required. 
Instead of using the `IBuildForDevice` interface, I suggest creating a new one that extends both IBuildForDevice and IRelease. The other option is to rename the current one, as the current properties are not required only for `IBuildForDevice` implementation.